transformer_engine/common/CMakeLists.txt

```diff
@@ -80,7 +80,7 @@ target_link_libraries(transformer_engine PUBLIC
                       CUDA::cuda_driver
                       CUDA::cudart
                       CUDA::nvrtc
-                      CUDA::nvToolsExt
+                      CUDA::nvtx3
```

Comment:
nvtx3 is a header-only library, so I think we don't need to add it to the target_link_libraries.